analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SecurityPInvokeMethodShouldNotBeCalled.cs

```diff
@@ -53,6 +53,10 @@ class Program
         public static void CoInitializeSecurity(int param) { } // Compliant non extern
         public extern void CoInitializeSecurity(string param); // Compliant non static
         public static extern int CoInitializeSecurity(int param1, string param2); // Compliant no DllImport
+        [DllImport(null)] // Error [CS0591]
+        public static extern int CoInitializeSecurity(int param1, string param2, string param3); // Compliant DllImport argument invalid
+        [DllImport("noOle32")]
+        public static extern int CoInitializeSecurity(int param1, string param2, string param3, string param4); // Compliant DllImport argument not "ole32"
```

Comment:
you have added these, but should these be added to `Main` as well? 

in my understanding, the rule should raise on the invocation?

Reply:
You are right. I was adding these to increase coverage and was wondering why it didn't increase. That solved that problem.